<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClass</class>
 <widget class="QMainWindow" name="MyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="tlLabel">
        <property name="text">
         <string>TL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tlLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="rateLabel">
        <property name="text">
         <string>Rate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rateLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="amountLabel">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="amountLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="usdButton">
        <property name="text">
         <string>USD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eurButton">
        <property name="text">
         <string>EUR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gbpButton">
        <property name="text">
         <string>GBP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cnyButton">
        <property name="text">
         <string>CNY</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>628</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCurrency_Converter">
    <property name="title">
     <string>Currency Converter</string>
    </property>
   </widget>
   <addaction name="menuCurrency_Converter"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
